components/admin_console/ldap_settings.jsx

```diff
@@ -83,6 +84,44 @@ export default class LdapSettings extends AdminSettings {
         );
     }
 
+    renderJobData(job) {
+        var matterMostUsers = '0';
+        var ldapUsers = '0';
+        var deleteCount = '0';
+        var updateCount = '0';
```

Comment:
Mind using `let` instead of `var? I'm surprised that eslint doesn't error on that

Reply:
will fix.